Framework/Algorithms/test/FindPeaksTest.h

```diff
@@ -224,52 +228,51 @@ class FindPeaksTest : public CxxTest::TestSuite {
+                  1.213020, 1.213510, 1.213990, 1.214480, 1.214970, 1.215450,
+                  1.215940, 1.216420, 1.216910, 1.217400, 1.217880, 1.218370,
+                  1.218860, 1.219350, 1.219830, 1.220320, 1.220810, 1.221300,
+                  1.221790, 1.222280, 1.222760, 1.223250, 1.223740, 1.224230,
+                  1.224720, 1.225210, 1.225700, 1.226190, 1.226680, 1.227170,
+                  1.227660, 1.228160, 1.228650, 1.229140, 1.229630, 1.230120,
+                  1.230610, 1.231110, 1.231600, 1.232090, 1.232580, 1.233080,
+                  1.233570, 1.234060, 1.234560, 1.235050, 1.235540, 1.236040,
+                  1.236530, 1.237030, 1.237520, 1.238020, 1.238510, 1.239010,
+                  1.239500, 1.240000, 1.240500, 1.240990, 1.241490, 1.241990,
+                  1.242480, 1.242980, 1.243480, 1.243970, 1.244470, 1.244970,
+                  1.245470, 1.245970, 1.246460, 1.246960, 1.247460, 1.247960,
+                  1.248460, 1.248960, 1.249460, 1.249960, 1.250460},
+        Counts{1619.0, 1644.0, 1616.0, 1589.0, 1608.0, 1612.0, 1630.0, 1671.0,
+               1588.0, 1577.0, 1616.0, 1556.0, 1625.0, 1655.0, 1552.0, 1539.0,
+               1538.0, 1542.0, 1558.0, 1628.0, 1557.0, 1606.0, 1563.0, 1611.0,
+               1584.0, 1447.0, 1532.0, 1580.0, 1539.0, 1513.0, 1601.0, 1558.0,
+               1567.0, 1573.0, 1551.0, 1465.0, 1602.0, 1543.0, 1538.0, 1515.0,
+               1556.0, 1574.0, 1519.0, 1452.0, 1568.0, 1522.0, 1518.0, 1603.0,
+               1538.0, 1659.0, 1685.0, 1763.0, 1846.0, 1872.0, 2018.0, 2035.0,
+               2113.0, 2131.0, 1921.0, 1947.0, 1756.0, 1603.0, 1602.0, 1552.0,
+               1558.0, 1518.0, 1512.0, 1511.0, 1466.0, 1474.0, 1368.0, 1463.0,
+               1447.0, 1409.0, 1381.0, 1478.0, 1445.0, 1429.0, 1447.0, 1354.0,
+               1430.0, 1440.0, 1423.0},
+        CountStandardDeviations{
+            40.236800, 40.546300, 40.199500, 39.862300, 40.099900, 40.149700,
+            40.373300, 40.877900, 39.849700, 39.711500, 40.199500, 39.446200,
+            40.311300, 40.681700, 39.395400, 39.230100, 39.217300, 39.268300,
+            39.471500, 40.348500, 39.458800, 40.074900, 39.534800, 40.137300,
+            39.799500, 38.039500, 39.140800, 39.749200, 39.230100, 38.897300,
+            40.012500, 39.471500, 39.585400, 39.661100, 39.382700, 38.275300,
+            40.025000, 39.281000, 39.217300, 38.923000, 39.446200, 39.673700,
+            38.974400, 38.105100, 39.598000, 39.012800, 38.961500, 40.037500,
+            39.217300, 40.730800, 41.048800, 41.988100, 42.965100, 43.266600,
+            44.922200, 45.111000, 45.967400, 46.162800, 43.829200, 44.124800,
+            41.904700, 40.037500, 40.025000, 39.395400, 39.471500, 38.961500,
+            38.884400, 38.871600, 38.288400, 38.392700, 36.986500, 38.249200,
+            38.039500, 37.536600, 37.161800, 38.444800, 38.013200, 37.802100,
+            38.039500, 36.796700, 37.815300, 37.947300, 37.722700,
```

Comment:
Those seem to be just the `sqrt` of the `Counts`, i.e., could probably be omitted?